e2e/pages/events/manageDocumentsEvent.page.js

```diff
@@ -2,8 +2,7 @@ const { I } = inject();
 const supportingDocumentsFragment = require('../../fragments/supportingDocuments.js');
 
 module.exports = {
-  supportingDocumentType: '',
-  fields: function(index, supportingDocumentType) {
+  fields: function(index) {
```

Comment:
Could the function be moved to only be on the `supportingDocuments` entry

Reply:
Yeah good idea! I'll do that.